docs/IDAM.md

```diff
@@ -0,0 +1,87 @@
+## Keycloak Package
```

Comment:
@UnicornChance should we move this to a different section of the docs/update the format to fit the docs site?